src/__tests__/signInAndOutHelper.test.ts

```diff
@@ -0,0 +1,59 @@
+import { handleLogoutClick } from "../helpers/signInAndOutHelper";
```

Comment:
@deepansh946 I highly suggest we keep test files colocated with their respective modules for ease of navigation and to easier spot what's tested and what is not.

Reply:
@deepansh946 I meant not moving the test folder but keeping the test files next to the modules they test so it's easy to spot which file has a corresponding test and which isn't without looking into nested folder.